src/lib/stitching/gravity/stitching.ts

```diff
@@ -126,6 +126,10 @@ export const gravityStitchingEnvironment = (
         viewingRoomsConnection(published: Boolean = true, first: Int, after: String, statuses: [ViewingRoomStatusEnum!]): ViewingRoomsConnection
       }
 
+      extend type Show {
+        viewingRoomsConnection(ids: [ID]): ViewingRoomsConnection
```

Comment:
`[ID!]!`?

Reply:
Actually this shouldn't have anything here, since this arg is injected in anyway.

Reply:
👍